draft-looker-jwm.xml

```diff
@@ -85,39 +85,39 @@
               or confidentiality via encryption for the JWM attribute set in similar ways to JWT for the JWT claim set.</t>
             </list>
           </t>
 
-          <t>JWMs also deviate in important ways from JWTs which prevents using the JWT specification directly to achieve the functionality provided by a JWM.</t>
+          <t>JWMs also deviate from JWTs in important ways that prevent a converged specification.</t>
             <t>
               <list style="symbols">
-                <t>JWTs due to the requirement of them being compact in nature and URL safe, mandate the usage of compact serialization for both 
-                JWS and JWE representations. This compact serialization requirement for JWS and JWE limits JWTs to only being able to feature a single 
-                digital signature and or encrypt a payload to a single recipient. Whereas JWMs do not require the use of compact serialization of JWE, 
-                instead they permit the JSON based serialization of JWS and JWE, which allows for a JWM to feature multiple digital signature or encrypted 
-                to multiple recipients.</t>
+                <t>JWM and JWT have different intents. A JWM is about a sender creating a message composed of attributes, where the message is destined for a
+                  recipient or recipients. Whereas a JWT is about an issuer expressing claims about a subject to an audience.</t>
 
-                <t>JWM and JWT have different intents. A JWM is about a sender creating a message composed of attributes, where the message is destined for a 
-                recipient or recipients. Whereas a JWT is about an issuer expressing claims about a subject to an audience.</t>
+                <t>The primary usage of JWTs centers around creating tokens that are digitally signed or integrity protected through a Message Authentication Code
+                  (MAC) by leveraging JWS. Encrypted JWTs via JWE are less common and as defined in Section 8 of <xref target="RFC7519"/>
+                  they are optional to implement. Whereas JWMs require both JWS and JWE implementations.</t>
 
-                <t>The primary usage of JWTs centers around creating tokens that are digitally signed or integrity protected through a Message Authentication Code 
-                (MAC) by leveraging JWS. Encrypted JWTs via JWE are less common and as defined in Section 8 of <xref target="RFC7519"/> 
-                they are optional to implement. Whereas JWMs require both JWS and JWE implementations.</t>
+                <t>Because JWTs must be compact and URL-safe, they require compact serialization for both
+                JWS and JWE representations. This means JWTs can feature only a single
+                digital signature, and/or encrypt for only a single recipient. In contrast, JWMs support multiple
+                digital signatures, and encryption for multiple recipients. They therefore
```

Comment:
I like the simplification of this paragraph, how about this as a further clarification?
  
.... "They achieve this through permitting the usage of the JSON-based serialization of JWS and JWE."

Reply:
Updated the verbiage to incorporate this suggestion.